Functions/Quantification/xASL_quant_FSL.m

```diff
@@ -75,7 +75,13 @@
     
     %% 3. Write the PWI as Nifti file for Basil/Fabber to read as input
     % FIXME would be good to have a brain mask at this point -> PM: if this would be a brainmask as well, we can skip creating a dummy input image here
+    
     PWI(isnan(PWI)) = 0;
+
+    BrainMask = xASL_io_Nifti2Im(x.P.Path_BrainMaskProcessing); % load the brain mask used for processing
+    PWI(BrainMask==0) = 0; % set voxels outside the mask to zero
```

Comment:
That's an incorrect approach IMHO. BASIL will have a specific input for brain mask. So this has to be passed as such to BASIL without trying to null the values ourselves.

Reply:
btw: this won't work with PWI4D that is passed to BASIL anyway - so the entire thing here has to be deleted.

Reply:
I see. So we need to provide an explicit mask to BASIL, not an implicit mask (which is what I did here)